src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("ForecastSummary", () => {
+  const validProps = {
```

Comment:
In this case, you are never using the `validProps` object as a whole so I would just define them as separate variables like this:
```
const searchText = "Edinburgh"
const setSearchText = jest.fn()
const onSubmit = jest.fn()
```
This would save you having to type `validProps.` everytime